package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "webpack-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "watch": {
+    "start": "{src}/*.js"
```

Comment:
This is good but let this also handle '.js' files inside sub-folders. 
You can change it to '{src}/**/*.js'

Reply:
Alright. thanks!